test/image/mocks/geo_across-antimeridian.json

```diff
@@ -0,0 +1,89 @@
+{
+    "data": [
+        {
+            "type": "scattergeo",
+            "lon": [170, 180, -180, -170],
+            "lat": [-50, -45, -10, -20.5],
+            "marker": { "size": 20 }
```

Comment:
for completeness, want to include lines here? Hard to imagine that they would do something funny like try to wrap the other way around the globe, but ya never know if it's not tested!

Reply:
Works fine with `mode: 'lines'`

![image](https://user-images.githubusercontent.com/6675409/30765330-25120e74-9fbd-11e7-8be0-2ef481be9386.png)

You scared me there, because I know `mapbox-gl` does the wrong thing in this case.

Reply:
done in https://github.com/plotly/plotly.js/pull/2030/commits/05e819429e76d6a49e770441d54086b05aebb8ed